app/controllers/concerns/part_of_claim_journey.rb

```diff
@@ -22,7 +22,7 @@ def send_unstarted_claimants_to_the_start
   end
 
   def skip_landing_page?
-    params[:skip_landing_page] == "true"
+    params.dig(:claim, :skip_landing_page) == "true"
```

Comment:
Given this module is included in the `SubmissionsController` is this safe to change here?
Do we need to copy the original implementation of the method into the `SubmissionsController`?